modules/netweaver/main.tf

```
@@ -60,26 +60,25 @@ resource "google_compute_attached_disk" "gcp_nw_attached_pd_0" {
   project     = "${var.project_id}"
   count       = "${var.usr_sap_size > 0 ? 1 : 0}"
   device_name = "${local.device_name_1}"
-  disk        = "${google_compute_disk.gcp_nw_pd_0.self_link}"
+  disk        = "${element(google_compute_disk.gcp_nw_pd_0.*.self_link, count.index)}"
```

`element` is not necessary for this type of expression in Terraform 0.12.

```suggestion
  disk        = google_compute_disk.gcp_nw_pd_0[count.index].self_link
```